platform/android/export/export.cpp

```diff
@@ -2747,6 +2838,30 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 			return ERR_CANT_CREATE;
 		}
 
+		List<String> copy_args;
+		String output_path;
+		if (export_bundle) {
+			if (p_debug) {
+				output_path = "android/build/build/outputs/bundle/debug/build_debug.aab";
+			} else {
+				output_path = "android/build/build/outputs/bundle/release/build_release.aab";
+			}
+		} else {
+			if (p_debug) {
+				output_path = "android/build/build/outputs/apk/debug/android_debug.apk";
+			} else {
+				output_path = "android/build/build/outputs/apk/release/android_release.apk";
+			}
+		}
+
+		copy_args.push_back(ProjectSettings::get_singleton()->get_resource_path().plus_file(output_path));
+		copy_args.push_back(p_path.replace("apk", (export_bundle ? "aab" : "apk")));
```

Comment:
Replace "apk" with "apk"? Maybe conditionally replace apk with aab if needed, something like this?

`copy_args.push_back(export_bundle ? p_path.replace("apk", "aab") : p_path);`